src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -36,7 +36,7 @@
 
     private String id;
 
-    private List<EnhancedAlert> enhancedAlerts = new ArrayList<>();
+    private List<String> activities = new ArrayList<>();
```

Comment:
The default activities are `RIDE, BOARD, EXIT`.

Reply:
Yes, those might be the defaults, but we're not reverting to the default values anywhere. We use whatever is in the json. If it's empty, then we're going to have the empty list, not the defaults.

Reply:
I was thinking more for the PB file which doesn't have these fields.

Reply:
Yep, fixed.